pkg/strc/middleware.go

```diff
@@ -64,12 +64,22 @@ type MiddlewareConfig struct {
 	// WithSpanID enables logging of the span ID. Defaults to false.
 	WithSpanID bool
 
+	// NoExtractTraceID disables extracting trace id from incoming requests. Defaults to false.
+	NoTraceContext bool
```

Comment:
A bit confusing to negate this, considering the other options are formulated positively. How about `WithTraceContext` ?

Reply:
What I have is idiomatic Go - I want the configuration to be usable by default without explicit initialization.